components/eam/cime_config/config_compsets.xml

```diff
@@ -291,64 +295,64 @@
 
     <entry id="RUN_STARTDATE">
       <values>
-	<value  compset="20TR_EAM">1850-01-01</value>
-	<value  compset="AMIP_EAM">1979-01-01</value>
-	<value  compset="HIST_EAM">1850-01-01</value>
-	<value  compset="PIPD_EAM">1850-01-01</value>
-	<value  compset="7905_EAM">1979-01-01</value>
-	<value  compset="5505_EAM">1955-01-01</value>
-	<value  compset="1996_EAM\d+%WCMX">1996-01-01</value>
-	<value  compset="2000_EAM\d+%WCMX">2000-01-01</value>
-	<value  compset="2000_EAM\d+%WCXI">2000-01-01</value>
-	<value  compset="GEOS_EAM\d+%WCCM">2005-01-01</value>
-	<value  compset="GEOS_EAM\d+%WTSM">1990-01-01</value>
-	<value  compset="GEOS_EAM\d+%TMOZ">2008-01-01</value>
-	<value  compset="GEOS_EAM\d+%TBAM">2008-01-01</value>
-	<value  compset="GEOS_EAM\d+%S">2005-01-01</value>
-	<value  compset="RCP[2468]_EAM\d+">2005-01-01</value>
-	<value  compset="C2R[68]_EAM">1950-01-01</value>
-	<value  compset="C2R4_EAM">2004-01-01</value>
-	<value  compset="AR95_EAM">1995-07-18</value>
-	<value  compset="AR97_EAM">1997-06-19</value>
+  <value  compset="20TR_EAM">1850-01-01</value>
```

Comment:
Looks like a bunch of indentation was removed that should not have been.

Reply:
Ah strange, I'll fix before merging.

Reply:
Oh, these are hard tabs in master @rljacob  it looks like someone's editor maybe removed them.

Reply:
As far as I can tell, tabs are ok in XML.  xmllint doesn't complain.  So leave them as tabs.

Reply:
Looks like my editor is the one that did this. No idea how though... I never touched this block

Reply:
Fixed.